resolwe/toolkit/docker_images/base/Dockerfile.fedora-31

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/fedora:31
+
+LABEL MAINTAINER Resolwe authors https://github.com/genialis/resolwe
+
+# XXX: Remove this after Resolwe learns how to mount re-* tools into every container
+COPY re-import.sh /re-import.sh
+# XXX: Remove this after Resolwe learns how to mount re-* tools into every container
+COPY curlprogress.py /usr/local/bin/
+
+
+RUN dnf install -y python-pip && \
+    pip3 install resolwe-runtime-utils && \
+    # XXX: Remove this hack after obsoleting re-checkrc
+    echo 're-checkrc() { _re-checkrc $? "$@"; }' >> /etc/profile.d/resolwe-base.sh && \
+    # XXX: Remove this hack after updating processes to no longer implicitly assume $NAME is set
+    # when calling re-import.sh
+    cat /re-import.sh >> /etc/profile.d/resolwe-base.sh && \
+    rm /re-import.sh && \
+    # XXX: Remove this after Resolwe learns how to mount re-* tools into every container
+    chmod +x /usr/local/bin/curlprogress.py && \
+    # Cleanup
+    dnf install -y --setopt=tsflags=nodocs zip && \
```

Comment:
This is not a part of the cleanup (as stated in the comment above). Is this package really required?

Reply:
The package is required, without it the docker image did not build. I will move it away from Cleanup section.